etc/profile-a-l/clipit.profile

```diff
@@ -34,16 +38,25 @@ nodvd
 nogroups
 noinput
 nonewprivs
+noprinters
 noroot
 nosound
 notv
 nou2f
 novideo
-protocol unix
+#protocol unix
```

Comment:
Why? Does it break something?

Reply:
Duh, forgot to uncomment it, which I did while testing. Can confirm it doesn't break anything. Pushing a fix shortly.